playbooks/openstack-caasp_instance.yml

```diff
@@ -163,27 +152,60 @@
             - skip_ansible_lint
 
         - name: Extend inventory in workspace with newly created caasp nodes
-          config_template:
-            src: "{{ socok8s_workspace }}/inventory/skeleton-inventory.yml"
+          copy:
+            content: "{{ caasp_overrides }}"
             dest: "{{ socok8s_workspace }}/inventory/caasp.yml"
-            config_type: yaml
-            config_overrides: "{{ caasp_overrides }}"
           vars:
-            caasp_overrides:
+            caasp_overrides: |
```

Comment:
wouldnt this be better stored as a template instead?

Reply:
That will be done in the step 3, when things will move to a role for DRY.

Reply:
:+1: